Automattic/wp-calypso
# Hosting Command Palette: Add/ssh commands

Related to https://github.com/Automattic/dotcom-forge/issues/4470

## Proposed Changes

This PR adds SSH commands that are mentioned in the issue above as indicated in the issue above. The new commands that are added are:

* Reset SSH/SFTP password (resets SFTP passwords and copies it to clipboard)
* Copy SSH connection string
* Open SSH details (redirects to Hosting Configuration page on AT sites)

## Testing Instructions

* Navigate to Calypso Live link or pull this branch
* Go to `/sites` page
* Press `cmd + k` on Mac or `ctrl + k` on Windows
* Search for `ssh`
* Try the SSH commands above and confirm you the data copied to the clipboard

<img width="1072" alt="Screenshot 2023-11-10 at 11 53 26 AM" src="https://github.com/Automattic/wp-calypso/assets/25575134/e0e62d21-df0b-472c-b483-5d91ddebc368">

**Notes**:

* At the moment, the command palette shows all the commands. I am wondering if we need to limit the number and display only some commands by default
* I am making an API request in `Copy SSH connection string` and `Reset SSH/SFTP password` to get the data that I need. It seems to be working relatively fast for me and it is the best approach that I found at the moment, but happy to reconsider if you think it leads to slowdown 
* Not sure if we need to limit the `Open SSH details` command only to Atomic sites or should keep it available for the sites on the Business plan

## Pre-merge Checklist
